.github/dependabot.yml

```diff
@@ -45,6 +45,9 @@ updates:
       # These are included with submariner-operator
       - dependency-name: github.com/submariner-io/admiral
       - dependency-name: github.com/submariner-io/submariner
+      # Operator Framework before 0.17.7 for k8s.io 0.26.x
+      - dependency-name: github.com/operator-framework/api
+        versions: ">= 0.17.7"
```

Comment:
We won't need this after the PRs to eliminate the `github.com/operator-framework/api` dependency are merged.